docs/git.rst

```diff
@@ -560,6 +560,15 @@ Again this applies:
 http://www.osnews.com/story/19266/WTFs_m
 
 
+The review-change-push-review cycle
+-----------------------------------
+
+After you have made a pull request, you should ask for a review via the GitHub interface. After the review you will frequently have to do some changes of your PR, perhaps add a missing changelog entry, and so on.
```

Comment:
Please follow the doc-guidelines, which here means here to use semantic linefeeds. See https://docs.plone.org/about/contributing/documentation_styleguide.html#line-length

Reply:
@jensens done:)